test/run-tests

```diff
@@ -532,7 +552,7 @@ def choose_task(gh, repos, images, config):
             commands = get_comment_commands(gh, owner, repo, number)
 
             for image in images:
-                status_context = f"{config['name']}/{image['name']}"
+                status_context = f"{config['name']}/{image['name']}/{ansible_ver_str}"
```

Comment:
The `status_context` is the string used in the PR to denote the status e.g. `linux-system-roles-test-staging/centos-7/ansible-2.9` - now the ansible version is added to every status.

Reply:
Will GitHub show such a long status properly?

Reply:
Yes, mostly - see https://github.com/linux-system-roles/kernel_settings/pull/20
```
 linux-system-roles-test-staging/fedora-32/ansible-2.9 Pending — localhost.localdomain@2020-07-01 14:35:38.813840 
```
In the UI, the timestamp is truncated, but it is also truncated for some of the other longer statuses

If length is a concern, note that most of the length of the status string comes from "linux-system-roles-test-" - so perhaps we could abbreviate that e.g. "lsr-test-" or ???

Reply:
> If length is a concern, note that most of the length of the status string comes from "linux-system-roles-test-" - so perhaps we could abbreviate that e.g. "lsr-test-" or ???

+1

Reply:
Since it being a status it is probably clear that it is a test run, so how about `system-role/ansible-2.9/fedora-32`. or `sysrole/ansible-2.9/fedora-32`? Actually, once could argue that `ansible-2.9/fedora-32` might be enough as well and maybe `ansible-2.9/fedora-32 (staging)` for the staging instance.

Reply:
ok - I will make this change in a separate PR

Reply:
> ok - I will make this change in a separate PR

Or is it ok to make this change in this PR?  I must change the status_context string in this PR in order for centos7 testing to work - maybe I should go ahead and drop the "linux-system-roles-test-" too?

Reply:
I'd like to double-check... If there's no word after linux-system-roles-test like this, how it's going to be displayed?
` linux-system-roles-test/rhel-x — linux-system-roles-39-s828j@2020-06-26 21:19:53.503386`

Reply:
> > ok - I will make this change in a separate PR
> 
> Or is it ok to make this change in this PR? I must change the status_context string in this PR in order for centos7 testing to work - maybe I should go ahead and drop the "linux-system-roles-test-" too?

Since you adjust the string it seems fine to me to drop the prefix, too.

Reply:
> I'd like to double-check... If there's no word after linux-system-roles-test like this, how it's going to be displayed?
> ` linux-system-roles-test/rhel-x — linux-system-roles-39-s828j@2020-06-26 21:19:53.503386`

This would be `rhel-x/ansible-xy — linux-system-roles-39-s828j@2020-06-26 21:19:53.503386`

Reply:
Thanks. So, `staging` is rather an exception. :)
+1

Reply:
and staging would be `rhel-x/ansible-x.y (staging) -- linux-system-roles-39-s828j....`

Reply:
Here is a real example from a real PR: https://github.com/linux-system-roles/kernel_settings/pull/20
`centos-8/ansible-2.9 (staging) Pending — localhost.localdomain@2020-07-01 18:19:15.258999`

Reply:
@tyll 
> Actually, once could argue that ansible-2.9/fedora-32 might be enough as well and maybe ansible-2.9/fedora-32 (staging) for the staging instance.

GitHub does not seem to sort the statuses (unfortunately), so whether to use `staging/image/ansible` or `image/ansible (staging)` to indicate the instance is more a matter of taste. Note though that this string is not just a UI/display string. It is used by the test-harness to identify what runs are pending or already completed.

Reply:
fixed in https://github.com/linux-system-roles/test-harness/pull/87/commits/a44492204e08443e57774e011be91e3436e71f77 - see https://github.com/linux-system-roles/kernel_settings/pull/20 for examples of what these new status strings look like